posthog/models/experiment.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Experiment(models.Model):
+    name: models.CharField = models.CharField(max_length=400, null=True, blank=True)
+    description: models.CharField = models.CharField(max_length=400, null=True, blank=True)
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
+    # Filters define the target metric of an Experiment
+    filters: models.JSONField = models.JSONField(default=dict)
+    parameters: models.JSONField = models.JSONField(default=dict, null=True)
+    feature_flag: models.ForeignKey = models.ForeignKey("FeatureFlag", blank=False, on_delete=models.CASCADE)
+    created_by: models.ForeignKey = models.ForeignKey("User", on_delete=models.CASCADE)
+    start_date: models.DateTimeField = models.DateTimeField()
```

Comment:
judging by @liyiy 's new frontend - I think `start_date` is nullable too? A null start date implies a draft experiment, one that hasn't been started yet.

Reply:
I think that makes sense and we won't have to add a separate attribute to mark whether it's draft or not :D

Reply:
Did you want the `start_date` to be passed in as a param? And I assume `end_date` will require some backend calculations?

Reply:
`end_date` would most probably be a button once the experiment is ongoing, if a user wants to end it now. Some UX to be figured out here (whether we want to default to a value based on calculations of how long the exp should run, but I'll defer this decision for now)

Reply:
`start_date`, good question. I guess when you click the "save and launch", you'd send the start_date param?

Otherwise I wouldn't be able to tell whether it's launching or not 👀